src/_lib/sass/src/_variables.scss

```diff
@@ -579,12 +579,12 @@ $modal-title-padding:         16px 32px !default;
 $modal-title-line-height:     $line-height-base !default;
 
 $modal-content-bg:                             $body-bg !default;
-$modal-content-border-color:                   get-color("gray", base) !default;
-$modal-content-fallback-border-color:          get-color("gray", base) !default;
+$modal-content-border-color:                   rgba($text-color, 0.075) !default;
```

Comment:
Is 0.075 correct?

Reply:
It's supposed to be 7.5% opacity, that's 0.075, right?

Reply:
Yep! Thanks